src/swift/devcontainer-template.json

```diff
@@ -3,16 +3,16 @@
 	"version": "1.1.0",
 	"name": "Swift",
 	"description": "Develop Swift based applications. Includes appropriate runtime args and everything you need to get up and running.",
-	"documentationURL": "https://github.com/swift-server/swift-dev-container/tree/main/README.md",
+	"documentationURL": "https://github.com/swift-server/swift-devcontainer-template/tree/main/README.md",
 	"publisher": "Swift Server Work Group",
-	"licenseURL": "https://github.com/swift-server/swift-dev-container/blob/main/LICENSE",
+	"licenseURL": "https://github.com/swift-server/swift-devcontainer-template/blob/main/LICENSE",
 	"options": {
 		"imageVariant": {
 			"type": "string",
 			"description": "Swift version:",
 			"proposals": [
 				"5.7",
-				"5.6-focal",
+				"5.6",
```

Comment:
Do we not want to keep focal to ensure building on ARM machines easily?

Reply:
I thought 5.6 on arm machines automatically chose focal

Reply:
No unfortunately not, `5.6` points to 5.6-bionic` which there are no ARM images for 😒

Reply:
Ok I'll revert that change